static/app/components/organizations/pageFilterRow.tsx

```diff
@@ -54,20 +57,17 @@ function PageFilterRow({
   );
 }
 
-const MultiselectCheckbox = styled(CheckboxFancy)`
-  position: relative;
+const checkboxInputStyles = css`
```

Comment:
How do you feel about just __always__ doing this, instead of allowing the hidden checkbox to be changed?

Reply:
Hmm, I guess this is pretty specific huh

Reply:
Always making the hitbox larger? I don't know about that, I think this is pretty specific to the use case here, since it needs to be about the same height as the dropdown item. Other use cases might want to make it a different size. But I understand this does seem icky.

I also think we'd need a different hover state if we had a larger hitbox, like the material ui checkboxes that show a circle around it. Without that it feels off
![CleanShot 2023-01-05 at 12 03 12](https://user-images.githubusercontent.com/10888943/210869858-fb5ce9b5-2f31-4626-a373-7e08a02e4ae7.png)

Reply:
Right right. Okay let's leave it as this for now